client/jetpack-cloud/sections/pricing/jpcom-masterbar/index.tsx

```diff
@@ -1,282 +1,285 @@
-import { Button, Gridicon } from '@automattic/components';
+import { Gridicon } from '@automattic/components';
+import { useLocale, jetpackComLocales } from '@automattic/i18n-utils';
 import classNames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
-import { useCallback, useMemo, useState } from 'react';
+import { useCallback, useMemo } from 'react';
 import * as React from 'react';
 import { useSelector } from 'react-redux';
 import ExternalLink from 'calypso/components/external-link';
+import Gravatar from 'calypso/components/gravatar';
 import JetpackLogo from 'calypso/components/jetpack-logo';
 import JetpackSaleBanner from 'calypso/jetpack-cloud/sections/pricing/sale-banner';
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { urlToDomainAndPath } from 'calypso/lib/url';
+import { isUserLoggedIn, getCurrentUser } from 'calypso/state/current-user/selectors';
 import { getJetpackSaleCoupon } from 'calypso/state/marketing/selectors';
+import useMobileBtn from './use-mobile-btn';
+import useSubmenuBtn from './use-submenu-btn';
+import useUserMenu from './use-user-menu';
+
 import './style.scss';
-import antiSpamIcon from './assets/icons/anti-spam.svg';
-import backupIcon from './assets/icons/backup.svg';
-import boostIcon from './assets/icons/boost.svg';
-import crmIcon from './assets/icons/crm.svg';
-import growthIcon from './assets/icons/growth.svg';
-import performanceIcon from './assets/icons/performance.svg';
-import scanIcon from './assets/icons/scan.svg';
-import searchIcon from './assets/icons/search.svg';
-import securityIcon from './assets/icons/security.svg';
-import videoIcon from './assets/icons/video.svg';
+
+export const MAIN_CONTENT_ID = 'content';
 
 const JETPACK_COM_BASE_URL = 'https://jetpack.com';
-const BP = 960; // Breakpoint defined in stylesheet
 
+/**
+ * WARNING: this component is a reflection of the Jetpack.com header, whose markup is located here:
+ * https://opengrok.a8c.com/source/xref/a8c/jetpackme-new/parts/shared/header.php
+ *
+ * Both headers should stay in sync as much as possible.
+ */
 const JetpackComMasterbar: React.FC = () => {
 	const translate = useTranslate();
-	const menuItems = useMemo(
+	const locale = useLocale();
+	const jetpackSaleCoupon = useSelector( getJetpackSaleCoupon );
+	const isLoggedIn = useSelector( isUserLoggedIn );
+	const user = useSelector( getCurrentUser );
+	const isJpComLocale = useMemo( () => jetpackComLocales.includes( locale ), [ locale ] );
+	const sections = useMemo(
 		() => [
 			{
 				label: translate( 'Products' ),
+				id: 'products',
 				items: [
 					{
-						category: {
-							label: translate( 'Security' ),
-							description: translate( 'Protect your site' ),
-							href: '/features/security/',
-							icon: securityIcon,
-						},
+						label: translate( 'Security' ),
+						tagline: translate( 'Protect your site' ),
+						href: `${ JETPACK_COM_BASE_URL }/features/security/`,
```

Comment:
Do you think keeping these URLs as they are without the added base URL might make it easier to test? For example, I'm currently using a jetpack cloud URL (eg. `https://container-goofy-brahmagupta.calypso.live/pricing`) and when navigating these links I'd expect it to stay on the `container-goofy-brahmagupta.calypso.live` site but instead it always goes to `jetpack.com`.